wordpress-mobile/WordPress-iOS
# Issue 7043 - 3D touch current site fix

Fixes #7043 

To test:
Sign in and verify the sites in the 3d touch shortcuts are correct
Change the site and verify the sites are correct again

@koke, any tips on how to unit test this?
